receiver/awscontainerinsightreceiver/internal/k8swindows/extractors/extractor.go

```diff
@@ -0,0 +1,24 @@
+package extractors
+
+import (
+	"time"
+
+	cextractor "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/cadvisor/extractors"
+	stats "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
+)
+
+// RawMetric Represent Container, Pod, Node Metric  Extractors.
+// Kubelet summary and HNS stats will be converted to Raw Metric for parsing by Extractors.
+type RawMetric struct {
+	Id          string
+	Name        string
+	Namespace   string
+	Time        time.Time
+	CPUStats    *stats.CPUStats
+	MemoryStats *stats.MemoryStats
+}
+
+type MetricExtractor interface {
```

Comment:
Can we implement similar to MetricExtractor interface which has method HasValue using the CPUstats?

 type MetricExtractor interface {
	HasValue(*cinfo.ContainerInfo) bool
	GetValue(info *cinfo.ContainerInfo, mInfo CPUMemInfoProvider, containerType string) []*CAdvisorMetric
	Shutdown() error
}